src/content/ui/adaptive-responsive/index.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Overview
+description: >
+  It's important to create an app,
+  whether for mobile or web,
+  that responds to size and orientation changes
+  and maximizes the use of each platform.
+short-title: Adaptive
+---
+
+One of Flutter's primary goals is to create a framework
+that allows you to develop apps from a single codebase
+that look and feel great on any platform.
+
+This means that your app might appear on screens of
+many different sizes, from a watch, to a foldable
+phone with two screens, to a high definition monitor.
+And your input device might be a physical or
+virtual keyboard, a mouse, a touchscreen, or
+any number of other devices.
+
+Two terms that describe these design concepts
+are _adaptive_ and _responsive_. Ideally,
+you'd want your app to be _both_ but what,
+exactly, does this mean?
+
+## What is responsive vs adaptive?
+
+An easy way to think about it is that responsive design
+is about fitting the UI _into_ the space and
+adaptive design is about the UI being _usable_ in
+the space.
+
+So, a responsive app adjusts the placement of design
+elements to _fit_ the available space. And an
+adaptive app selects the appropriate layout and
+input devices to be usable _in_ the available space.
+For example, on the current device,
+should the UI use bottom navigation or
+side-panel navigation?
+
+:::note
+Often adaptive and responsive concepts are
+collapsed into a single term. Most often,
+_adaptive design_ is used to refer to both
+adaptive and responsive.
+:::
+
+This section covers various aspects of adaptive and
+responsive design:
+
+* Create a responsive [Layout][]
+* Handle [Input][] for adaptive apps
+* Consider [Accessibility][] for responsive apps
+* Respond to [Large screen][] devices
+* Consider [Platform idioms][] for adaptive apps
+  to make your app feel more native on a particular device
+* Use [Best practices][] for responsive and adaptive apps,
+  including **recommendations for large screens**
+
+[Accessibility]: /ui/adaptive-responsive/accessibility
+[Best practices]: /ui/adaptive-responsive/best-practices
+[Input]: /ui/adaptive-responsive/input
+[Large screen]: /ui/adaptive-responsive/large-screens
+[Layout]: /ui/adaptive-responsive/layout
+[Platform idioms]: /ui/adaptive-responsive/idioms
+
+## Examples
+
+If you'd like to see how these concepts come together,
+check out the [Flokk][] and [Folio][] examples.
+
+[Flokk]: {{site.github}}/gskinnerTeam/flokk
```

Comment:
Do we have a policy against linking to gskinner code? if not then I think we should link to the gskinner wonderous app. https://github.com/gskinnerTeam/flutter-wonderous-app

Reply:
+1 we should remove the flokk and folio examples completely. they are not being updated anymore and wonderous is our new example. there should be no issue linking it

Reply:
Done.